tools/nl/embeddings/build_custom_dc_embeddings.py

```diff
@@ -95,9 +96,10 @@ def download(embeddings_yaml_path: str):
       embeddings_yaml_path, default_ft_embeddings_info)
 
 
-def build(model_version: str, sv_sentences_csv_path: str, output_dir: str):
```

Comment:
Can you run these 2 commands to ensure the script runs as expected:

```
build_custom_dc_embeddings.py --mode=download --embeddings_yaml_path=tmp/embeddings.yaml
```

Once this runs successfully, check the generated `tmp/embeddings.yaml` and validate it conforms to the new format.

> Login to gcloud before running the next one.

```
python3 build_custom_dc_embeddings.py \                                                      
--sv_sentences_csv_path=gs://datcom-floret-dev-resources/temp/nl/sentences.csv \
--output_dir=tmp
```

Check that it generates an embeddings CSV in the `tmp` folder.

> You may need to create a `tmp` folder before running these commands.

Reply:
thanks for the pointer, scripts run as expected

outputs
/tmp/embeddings.yaml: https://paste.googleplex.com/6011620073668608
/tmp/custom_embeddings.yaml: https://paste.googleplex.com/4865867779407872